yamllint/rules/document_start.py

```diff
@@ -94,6 +98,26 @@ def check(conf, token, prev, next, nextnext, context):
             yield LintProblem(token.start_mark.line + 1, 1,
                               'missing document start "---"')
 
+        if (conf['max-empty-lines-after'] >= 0 and
+                isinstance(prev, yaml.DocumentStartToken)):
+
+            empty_lines = token.start_mark.line - prev.start_mark.line - 1
```

Comment:
Suggestion: put this line above the `if` block, so we don't have to duplicate it on line 114.
Bonus: combine multiple `if` blocks (lines 101 + 106) into one, if it's more readable (I'm not sure...)

Reply:
I've shuffled the `if`s around until it looked good :smile: